README.md

```diff
@@ -0,0 +1,20 @@
+﻿## Testing
+
+### Prerequisites
+
+-   node 12.20.0
+-   yarn 1.22.0
+
+### Installation
+
+-   `yarn install`
+
+### Verification
+
+-   Verifying all the contracts:
+    -   `yarn test` (quick testing)
+    -   `yarn coverage` (full coverage)
+
+## License

```

Comment:
no need for this section (sticking to MIT)

Reply:
Removed.